Build `/data/postgresql/support` page

jmuzina opened the pull request:
## Done

- Builds the `/data/postgresql/support` page per [copy doc](https://docs.google.com/document/d/1scPktUiI2XRVoYcR4L6qUZ91iZ0upZA-FaqyJN7uMJg/edit?tab=t.0) and [figma design](https://www.figma.com/design/uJUwbUAGDog4KCMivrMpnl/24.10-Data-Fabric?node-id=2116-3102&node-type=frame&m=dev)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- [List additional steps to QA the new features or prove the bug has been resolved]

## Issue / Card

Fixes [WD-15564](https://warthogs.atlassian.net/browse/WD-15564)

## Screenshots
<img width="1225" alt="Screenshot 2024-10-23 at 3 03 53 PM" src="https://github.com/user-attachments/assets/6b9235dc-940e-4b3b-9b81-d9fcbee4aa8e">

webteam-app commented:
### [<img src='https://assets.ubuntu.com/v1/6baef514-ubuntu-circle-of-friends-large.svg' height=32 width=32> Demo</img>](https://canonical-com-1406.demos.haus)
### [<img src='https://assets.ubuntu.com/v1/e512b0e2-jenkins.svg' height=32 width=32> Jenkins </img>](https://jenkins.canonical.com/webteam/job/start-demo/1204)
### [<img src='https://assets.ubuntu.com/v1/7144ec6d-logo-jaas-icon.svg' height=32 width=32> demos.haus </img>](https://demos.haus)

jmuzina commented:
The PostgreSQL consulting image is currently not using the image template due to an error I am encountering with the image template. 

See [MM](https://chat.canonical.com/canonical/pl/f7uox7cf3bno7y8wxmjybr7naw) for details

jmuzina commented on templates/data/postgresql/support.html:
```diff
@@ -0,0 +1,635 @@
+https://docs.google.com/document/d/1scPktUiI2XRVoYcR4L6qUZ91iZ0upZA-FaqyJN7uMJg/edit
+{% endblock meta_copydoc %}
+
+{% block title %}Get comprehensive support for PostgreSQL, the world's most acclaimed open source database management system.{% endblock %}
```
This title may change, it is longer than the 60 chars recommendation in the copy doc template ([comment ref](https://docs.google.com/document/d/1scPktUiI2XRVoYcR4L6qUZ91iZ0upZA-FaqyJN7uMJg/edit?disco=AAABT_iI3g4))

jmuzina commented on templates/data/postgresql/support.html:
```diff
@@ -0,0 +1,635 @@
+    <div class="col">
+      <h2>PostgreSQL consulting</h2>
+      <div class="p-image-container--16-9 is-highlighted">
+        <img class="p-image-container__image" src="https://assets.ubuntu.com/v1/7bf5c40d-Canonical%20OS%20Infrastructure%20Illustrations%20FINAL_Hardware%20Ecosystem.svg" width="600" alt="Canonical hardware ecosystem"/>
```
The PostgreSQL consulting image is currently not using the image template due to an error I am encountering with the image template. 

See [MM](https://chat.canonical.com/canonical/pl/f7uox7cf3bno7y8wxmjybr7naw) for details

jmuzina replied:
Fixed

mattea-turic commented:
- `<br>` after "canonical" in the spotlight header pls
- and another `<br>` after "our" in the link under the "Managed Postgresql" comp pls
- you can safely remove the 24px padding here as we only need the 64px you've already added to the whole section
<img width="500" alt="Screenshot 2024-10-24 at 11 25 22" src="https://github.com/user-attachments/assets/1b710d48-7045-407d-9b27-87e89c5c9a22">

- Apologies for this, but I made a design change here as it didn't look right to me. So rather than having the image to the left there, I've instead placed all the elements vertically. 
<img width="500" alt="Screenshot 2024-10-24 at 11 26 24" src="https://github.com/user-attachments/assets/e35419a2-15e1-48aa-b0d0-31e6aa9994f4">
<img width="500" alt="Screenshot 2024-10-24 at 11 27 22" src="https://github.com/user-attachments/assets/1360d151-1724-452c-890b-4c731b3e71ef">

- There is also reg. bottom padding missing from that section, as well as in the sections "PostgreSQL training", "Come for PostgreSQL, get security and support for your entire stack", and the logo section
- Postgresql training is using the wrong asset (this might be asset manager's fault for not making it clear what's deprecated tbf), so pls use [this one](https://assets.ubuntu.com/v1/8135cf37-training.png)
- For the logo section, I've updated the assets as the logos should've been smaller (my bad), so if you could use those pls; I've added the suffix '-container-vert-fill' to them and they're in [this server](https://assets.ubuntu.com/manager?tag=spark-support-assets)
- Again, the shallow padding here isn't needed; only the 64px for the entire section. The other list items can go without the bottom padding too, as in the figma
<img width="500" alt="Screenshot 2024-10-24 at 11 32 19" src="https://github.com/user-attachments/assets/f09862dc-54d0-4bcc-989d-3227b8189078">

- and one more thing, the last section should have 128px (deep) bottom padding

Thanks in advance for the changes! :)

jmuzina commented:
>  the last section should have 128px (deep) bottom padding

@mattea-turic 

Thanks for pointing this out - I copied this from the [managed PostgreSQL page](https://canonical.com/data/postgresql/managed) which does not have the deep bottom padding. Should it be changed on that page?

Trademark section on the managed page:
<img width="1149" alt="Screenshot 2024-10-24 at 12 56 53 PM" src="https://github.com/user-attachments/assets/1840f7bb-54fa-47d2-bde4-6442a9cdea3c">

Trademark section on the support page, after applying the deep shallow section spacing:
<img width="1085" alt="Screenshot 2024-10-24 at 1 00 54 PM" src="https://github.com/user-attachments/assets/fc1d321e-ae8f-47ff-b404-51376c707c08">

I believe I've addressed all of your feedback. Have a look and let me know what you think, thanks for the review :)

mattea-turic commented:
Looks great, thanks!

There's just one more thing, but I'll +1 since it's such a tiny change:

Just adding a shallow padding to the text before the cta:
<img width="500" alt="Screenshot 2024-10-25 at 13 06 02" src="https://github.com/user-attachments/assets/18b475b6-7057-49c8-a09c-606e9a33c648">
<img width="500" alt="Screenshot 2024-10-25 at 13 05 26" src="https://github.com/user-attachments/assets/f946d369-0b95-417c-a593-bcff8ed206d6">

juanruitina commented:
I see the logo in the hero looks different in every PostgreSQL page, @mattea-turic can you assist?

In the "Come for PostgreSQL, get security and support for your entire stack", I'd suggest to implement the links in a similar way as we did in the [Valkey support page](https://canonical.com/data/valkey), so the logos are also clickable: `a` elements wrapping logo and link, with an `aria-label` with the name of the database.

The H5 headings should be H3s instead (but preserve the H5 styling): `<h3 class="p-heading--5">Example</h3>`. We do this to keep a clean heading tree for good accessibility.

mattea-turic commented:
> I see the logo in the hero looks different in every PostgreSQL page, @mattea-turic can you assist?
> 

Just added [this one](https://assets.ubuntu.com/v1/1ff056bf-postgres-logo.png), that is used in the /support and /managed figma files. The /what-is-postgresql page was done a while back, so that's why it's inconsistent, but we can apply the same one

Thanks for pointing that out @juanruitina

jmuzina commented:
Thanks @juanruitina , I've implemented your feedback, have another look and let me know what you think!

juanruitina commented:
UX+1, thank you for the tweaks!

akbarkz commented on templates/data/postgresql/support.html:
```diff
@@ -0,0 +1,731 @@
+  <div class="row--25-75-on-large">
+    <div class="col">
+      <div class="p-section--shallow">
+        {{ image (
```
please remove the space between "image" and the bracket and format the file with djLint

jmuzina replied:
Fixed

akbarkz commented on templates/data/postgresql/support.html:
```diff
@@ -0,0 +1,731 @@
+
+      <div class="p-equal-height-row__item">
+        <hr class="p-rule--highlight u-hide--medium u-hide--small" />
+
```
please remove the empty line and in similar places below

jmuzina replied:
Fixed

akbarkz commented on templates/data/postgresql/support.html:
please fix the closing tag here and in several places below. Looks like you're using a formatter that is not djLint

jmuzina replied:
Yes, it looks like Prettier output some things strangely. I've adjusted it, phal

akbarkz commented on templates/data/postgresql/support.html:
redundant empty line?

jmuzina replied:
Fixed
